shopfloor/services/cluster_picking.py

```diff
@@ -416,7 +417,7 @@ def unassign(self, picking_batch_id):
             batch.write({"state": "draft", "user_id": False})
         return self._response_for_start()
 
-    def scan_line(self, picking_batch_id, move_line_id, barcode):
+    def scan_line(self, picking_batch_id, move_line_id, barcode, sublocation_id=None):
```

Comment:
```suggestion
    def scan_line(self, picking_batch_id, move_line_id, barcode, package=None):
```

Reply:
After discussion we do not need to pass the package between the frontend and backend.
But I still believe we need to do this for the sublocation, example :

> The user scan a location containing multiple products. He will first scan the location but the current implementation will ask him to scan the product. And when the scan line or packaging first option is enabled the backend needs to remember that a location was scanned previously

.